server/portal/apps/projects/workspace_operations/shared_workspace_operations.py

```diff
@@ -83,8 +83,10 @@ def submit_workspace_acls_job(
     client = service_account()
     portal_name = settings.PORTAL_NAMESPACE
 
+    prj = user_client.systems.getSystem(systemId=system_id)
```

Comment:
Could we also grab this with the service account client?

Reply:
I thought we would be able to, but when a project is created in the portal, the service account doesn't have any permissions on it. `SYSLIB_UNAUTH Permission denied.` 

I could refactor the system creation process to add the service account as a member of every project, but I didn't think that was true to the Tapis V3 design principles.

Reply:
Ah right, I had thought the project was created by the admin client.